pkg/server/event_handler_test.go

```diff
@@ -176,18 +211,20 @@ func TestEventHandler_Handle(t *testing.T) {
 	t.Parallel()
 
 	cases := []struct {
-		name             string
-		notification     pubsub.Message
-		gcsObjectBytes   []byte
-		processors       []Processor[*structpb.Struct]
-		successMessenger Messenger
-		failureMessenger Messenger
-		wantErrSubstr    string
+		name              string
+		notification      pubsub.Message
+		gcsObjectBytes    []byte
+		githubSourceBytes []byte
+		processors        []Processor[*structpb.Struct]
+		successMessenger  Messenger
+		failureMessenger  Messenger
+		wantErrSubstr     string
 	}{
 		{
 			name: "success",
 			notification: pubsub.Message{
 				Attributes: map[string]string{"bucketId": "foo", "objectId": "bar"},
+				Data:       getFakeMetadata(),
```

Comment:
This is added but I don't see we validate it in the result (in the successMessenger). Or we didn't do that in the first place, which we then should fix (could be a follow up)?

Reply:
Created #97, will do this as a follow up.

Reply:
If it's not used, should we defer adding this?

Reply:
`Data` in `notification` has the githubSourceMetadata which will be used in [parseGitHubSource](https://github.com/abcxyz/pmap/blob/add_pmap_event_info/pkg/server/event_handler.go#L324).